inference-engine/tests/functional/plugin/cpu/single_layer_tests/space_to_batch.cpp

```diff
@@ -2,77 +2,98 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include <shared_test_classes/single_layer/space_to_batch.hpp>
+#include "shared_test_classes/base/ov_subgraph.hpp"
 #include "ngraph_functions/builders.hpp"
 #include "test_utils/cpu_test_utils.hpp"
 
 using namespace InferenceEngine;
 using namespace CPUTestUtils;
 using namespace ngraph::opset3;
+using namespace ov::test;
 
 namespace CPULayerTestsDefinitions  {
 
-typedef std::tuple<
-        LayerTestsDefinitions::spaceToBatchParamsTuple,
-        CPUSpecificParams> SpaceToBatchLayerCPUTestParamSet;
+using SpaceToBatchLayerTestCPUParams = std::tuple<
+        std::vector<InputShape>,            // Input shapes
```

Comment:
as I can see in this test we have dynamic shapes only on `data`
so looks like enough `InputShape` instead vector